src/components/Footer.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <v-footer color="deep-purple-lighten-4">
+        <v-col class="text-center d-flex align-center justify-center">
+            <v-icon class="mr-3">
+                <img src="@/assets/logo-aldebaram.svg" alt="Icon" class="icon-size">
+            </v-icon>
+            <span>&copy; {{ new Date().getFullYear() }} Terravision by Aldebaram</span>
+        </v-col>
+    </v-footer>
+</template>
+
+<script>
+export default {
+    name: 'FooterComponent',
+}
+</script>
```

Comment:
```suggestion
<script setup lang="ts"></script>
```
A utilização dessa forma foi utilizada mais para facilitar a migração das equipes do vue2 para o vue3, mas a padronização (e como o mineda está passando) é utilizar [https://vuejs.org/api/sfc-script-setup](script-setup).